marathon/src/main/scala/io/buoyant/marathon/v2/AppIdNamer.scala

```diff
@@ -70,15 +70,15 @@ class AppIdNamer(
             state() = Activity.Ok(apps)
             Trace.recordBinary("marathon.apps", apps.map(_.show).mkString(","))
             if (!stopped) {
-              pending = Future.sleep(ttl).onSuccess(_ => loop())
+              pending = Future.sleep(ttl.head).onSuccess(_ => loop())
```

Comment:
It looks like we're just always using `ttl.head` which will only use the first value of the stream over and over.  I think we probably want to modify `loop` to accept a stream and sleep based on the head of the stream and pass the tail of the stream into the recursive call to `loop`.